pkgs/development/haskell-modules/with-packages-wrapper.nix

```diff
@@ -46,14 +47,18 @@ let
   llvm          = lib.makeBinPath
                   ([ llvmPackages.llvm ]
                    ++ lib.optional stdenv.isDarwin llvmPackages.clang);
+  ghcLibdirLink = runCommand "ghc_libdir" {} ''
+    mkdir -p '${libDir}'
+    echo '${ghcLibdir}' > '${libDir}/ghc_libdir_override'
```

Comment:
`grep`ing the ghcjs source only shows `ghc_libdir`, so I suspect this was merely intended to plumb the path over in `mv '${libDir}/ghc_libdir_override' '${libDir}/ghc_libdir'`.